cpp/tests/utilities/column_utilities.cu

```diff
@@ -33,18 +33,33 @@
 namespace cudf {
 namespace test {
 
-// Property equality
-void expect_column_properties_equal(cudf::column_view lhs, cudf::column_view rhs) {
+namespace {   // anonymous
+
+// check for semantic property equality.  the difference between this and 
```

Comment:
I'm not sure about the name, because nulls can be part of the semantics too, can't they? How about just having `expect_column_properties_equal` with a bool parameter to specify whether to compare nullability and null count?

Reply:
That works too.  I assume the preference would be a default value of true to preserve old behavior?

Reply:
Yeah.